Example mod/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("2.13.5")]
-[assembly: AssemblyFileVersion("2.13.5")]
+[assembly: AssemblyVersion("2.14.0.1")]
+[assembly: AssemblyFileVersion("2.14.0.1")]
```

Comment:
I could be mistaken, but I thought we had decided we didn't want to use the revision number anymore?
@PrimeSonic @MrPurple6411 Can confirm?

```suggestion
[assembly: AssemblyVersion("2.14.0")]
[assembly: AssemblyFileVersion("2.14.0")]
```

Reply:
Version bumped for a modder to test the build pre-release. Should go back to 2.14 once it's ready for merge.

Reply:
My concern with this approach is that it's easy to accidentally forget to drop the revision number. What's in the PR itself should be what you want merged into master. If you want to make testing builds with different assembly versions, fine, but don't commit that to the PR because it's easy to end up merging in temporary changes.

See the fact that you forgot to revert a change in this PR for an example of how easy it is :P